### DISCUSSION TITLE
Markdown Tables

### DISCUSSION THREAD

**Discussion by calebstewart** [Textualize/rich#1254] [Q&A] [answered]
Maybe I missed something somewhere, but my tables aren't being rendered. They are being shown as one long line (still with the pipes). My table looks like this:

```
| This | Would |
| ------ | -------- |
| Be | My |
| Table | Data |
```

But it is being rendered as:

```
| This | Would | | ------ | -------- | | Be | My | | Table | Data |
```

Could markdown table support be added? I would think it would be (relatively) easy to add, since the Rich Tables already exist with many (if not all) of the features of standard markdown tables. If I've glossed over something in the documentation, feel free to set me straight. I'd love to hear I'm wrong :stuck_out_tongue_closed_eyes: 

## Edit

This is apparently a deficiency in the upstream `commonmark` library which refuses to support any flavor of markdown tables, because *OuR lOrD aND sAViOUr JOhn gRUbEr* didn't specify it in the original definition. [For reference, here's the upstream discussion](https://talk.commonmark.org/t/tables-in-pure-markdown)

**Comment by willmcgugan** [comment, marked as answer]
There is another markdown lib which is more actively developed. At some point I will switch to the new parser, but until then Markdown tables aren't supported I'm afraid.
